normandy/health/api/views.py

```diff
@@ -28,3 +32,46 @@ def version(request):
         'source': 'https://github.com/mozilla/normandy',
         'commit': get_commit(),
     })
+
+
+@api_view(['GET'])
+def health(request):
+    include_deployment_checks = not settings.DEBUG
+    all_checks = checks_registry.get_checks(include_deployment_checks)
+
+    details = {check.__name__: health_check_detail(check) for check in all_checks}
+
+    counts = Counter()
+    for detail in details.values():
+        counts[detail['status']] += 1
+
+    if all(detail['status'] == 'ok' for detail in details.values()):
```

Comment:
It seems a little harsh to fail on INFO and DEBUG messages. Or are even those messages we don't want to ignore?